src/frontend/Komponenter/Behandling/Simulering/Tilbakekreving.tsx

```diff
@@ -123,7 +127,7 @@ export const Tilbakekreving: React.FC<TilbakekrevingProps> = ({ behandlingId })
             url: `/familie-ef-sak/api/tilbakekreving/${behandlingId}`,
             data: {
                 valg: tilbakekrevingsvalg,
-                varseltekst: varseltekst,
+                varseltekst: inkluderVarseltekst(tilbakekrevingsvalg) ? varseltekst : '',
```

Comment:
Hadde det vært bedre å nulle ut varseltekst i radio-onchange i `Tilbakekrevingskjema.tsx`? Da sørger vi iallefall for at staten er helt uten varseltekst før man evt. sender inn?

Reply:
Eller inni `endreTilbakekrevingsvalg`, er kanskje bedre...